src/Component.php

```diff
@@ -45,7 +45,7 @@ protected function run(): void
                 ->setIncremental(true)
                 ->setPrimaryKeyColumns(['id']);
 
-            $this->getManifestManager()->writeTableManifest('profiles', $outTableManifestOptions);
+            $this->getManifestManager()->writeTableManifest('profiles.csv', $outTableManifestOptions);
```

Comment:
@ondrajodas Dopln prosim test, kde to bez tejto zmeny bude padat.

Reply:
doplnil jsem tady:
https://github.com/keboola/google-analytics-extractor/pull/52/commits/80e025b6c406a2353e6d0cfa70a04ec4f9c9d422

budou se vyhledávat manifesty končící `.csv.manifest` jinak se ani manifesty nesmí jmenovat

Reply:
a to bez https://github.com/keboola/google-analytics-extractor/pull/52/commits/2c5b5ac933c595b03a6fdef1989248c64f28495c neprojde testama